scalameta/scalameta/src/test/scala/scala/meta/tests/prettyprinters/SyntacticSuite.scala

```diff
@@ -184,6 +184,27 @@ class SyntacticSuite extends scala.meta.tests.parsers.ParseSuite {
     assert(Dotty(q"trait T(a: Int)").syntax === "trait T(a: Int)")
   }
 
+  test("literalTypes") {
+    intercept[ParseException] {
+      dialects.Scala211("val a : 42 = 42").parse[Stat].get.show[Syntax]
+    }
+    val Scala211 = null // TODO: #389
+    import scala.meta.dialects.Dotty
+    assert(q"val a: 42 = 42".show[Syntax] === "val a: 42 = 42")
```

Comment:
OK, thanks for checking. I didn't know it wasn't supported, just saw it in the contributors post